testlib.h

```diff
@@ -63,6 +63,7 @@
  */
 
 const char *latestFeatures[] = {
+        "rnd.distict(size, n) and rnd.distict(size, from, to)",
```

Comment:
Shouldn't it be spelt as `distinct` everywhere?

Reply:
Thanks, fixing it.